common/src/main/proto/DataRequestMessages.proto

```diff
@@ -0,0 +1,32 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option java_package = "org.apache.celeborn.common.protocol";
+option java_multiple_files = true;
+
+enum DataRequestMessageType {
```

Comment:
We may better call this one `DataMessageType` to align with `ControlMessageType`, but separate the DataRequestMessage, since there is a corresponding ResponseMessage

Reply:
Is `ControlMessageType` means that MessageType in TransportMessages.proto?

Reply:
yea, I believe that's the original principle for how to divide the message.

Reply:
Make sense, I think `DataRequestMessage` should be renamed to `DataMessage` because there might be response and renaming will be trouble if various messages had changed.

Reply:
Due to transport messages containing return messages, the data message might be expanded.